cmd/commands/image.go

```diff
@@ -90,7 +90,7 @@ func ImageRelocate(c *core.ImageClient) *cobra.Command {
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			err := (*c).RelocateImages(options)
+			err := (*c).RelocateImages(options, cmd.OutOrStdout())
```

Comment:
Would be better to inject the Writer into `imageClient` in `wiring.go` IMO, to be consistent with other signatures

Reply:
Yes, of course, good idea. I'll look into that now.

Reply:
Fixed.